MantidPlot/src/Mantid/MantidMatrixModel.cpp

```diff
@@ -276,3 +293,31 @@ bool MantidMatrixModel::checkMonitorCache(int row) const {
     return false;
   }
 }
+
+/**   Checks the row cache to see if the detector flag is stored, then returns
+it, otherwise it looks it up and adds it to the cache for quick lookup
+@param row :: current row in the table that maps to a detector.
+@return bool :: the value of if the detector is masked or not.
+*/
+bool MantidMatrixModel::checkMaskedCache(int row) const {
```

Comment:
Not necessarily, I just tried to follow exactly what was there for monitor check few lines above.
I think this does not matter much, since it is a private method anyway.